### DISCUSSION TITLE
Contributing a tutorial

### DISCUSSION THREAD

**Discussion by ebrahimebrahim** [Project-MONAI/tutorials#452] [General] [answered]
I'd like to contribute a tutorial demonstrating the use of MONAI for simultaneous training of a registration and a segmentation network, following the approach of [DeepAtlas](https://arxiv.org/abs/1904.08465).

Is there anything I need to know about contributing MONAI tutorials? I have already seen the [note to developers](https://github.com/Project-MONAI/tutorials#3-note-to-developers) in the readme. Anything else I should know while I finalize the tutorial notebook?

**Comment by Nic-Ma** [comment, marked as answer]
Hi @ebrahimebrahim ,

Thanks very much for your contribution.
I think you can just refer to some existing tutorial notebook, like:
https://github.com/Project-MONAI/tutorials/blob/master/3d_registration/paired_lung_ct.ipynb

Thanks.

**Reply by ebrahimebrahim** [reply]
Great, thank you!
